### ISSUE TITLE
About training/validation losses

### ISSUE THREAD

**Issue by SURABHI-GUPTA** [browatbn2/3FabRec#20]
@browatbn2 Hey.
For only the unsupervised reconstruction task, have you tried looking for the training/validation losses? Just plotting the reconstruction loss for all the three steps (recon, with gan, with gan & with ssim) the losses don't seem to converge and are jittery, although the reconstruction quality improves. Can you comment on this behavior of loss curves?

**Comment by browatbn2** [comment]
Hi, if you're training without the gan, the recon loss should keep going down if the reconstruction quality improves. It just measures L1(L2) distance between input and output images. 
In the other cases the losses compete and image quality improves with the reconstruction loss staying steady or even going up a bit.
